src/pages/Landing/Landing.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import { MdKeyboardArrowRight } from "react-icons/md";
+import { HomeConst } from "../../core/constants/home";
+import Audio from "../../components/audio/Audio";
+import bg from "../../assets/images/bg.svg";
+import FakeList from "./MicroComponents/FakeList";
+import Spinner from "./MicroComponents/Spinner";
+import ListSkeleton from "./MicroComponents/ListSkeleton";
+import "./style.css";
+
+
+function Landing() {
+
+  const [fakeLoader, setFakeLoader] = useState<Boolean>(false);
```

Comment:
Loader cannot be fake, but the variables or logic can be mocked. Please rename this accordingly